emergence_lib/src/player_interaction/tile_selection.rs

```diff
@@ -0,0 +1,238 @@
+//! Selecting tiles to be built on, inspected or modified
+
+use bevy::{prelude::*, utils::HashSet};
+use bevy_ecs_tilemap::tiles::{TilePos, TileVisible};
+use leafwing_input_manager::{
+    prelude::{ActionState, InputManagerPlugin, InputMap},
+    user_input::{InputKind, Modifier, UserInput},
+    Actionlike,
+};
+
+use super::cursor::CursorTilePos;
+
+/// Actions that can be used to select tiles.
+///
+/// If a tile is not selected, it will be added to the selection.
+/// If it is already selected, it will be removed from the selection.
+#[derive(Actionlike, Clone)]
+pub enum TileSelectionAction {
+    /// Selects a single tile.
```

Comment:
...and deselects all other tiles

Reply:
I corrected this documentation in an upcoming commit.